src/lib/libast/features/time

```diff
@@ -1,6 +1,7 @@
 lib	getrusage,nanosleep,usleep,_strftime
 typ	clock_t = uint32_t
-typ	time_t = uint32_t
+typ	time_t = uint64_t
```

Comment:
Since `time_t` is unsigned on many systems, I think we should default to a signed `int64_t` here so that we can store times before the 1st of January 1970.